bee-node/bee-node/config.chrysalis-mainnet.toml

```diff
@@ -137,3 +137,8 @@ session_timeout = 86400
 user            = "admin"
 password_salt   = "0000000000000000000000000000000000000000000000000000000000000000"
 password_hash   = "0000000000000000000000000000000000000000000000000000000000000000"
+
+[tracing]
+console_enabled = false
+flamegraph_enabled = false
+flamegraph_output_path = "./flamegraph"
```

Comment:
Maybe this should be `flamegraph_output_dir`?

Reply:
I was thinking the same thing when I was writing it, but ended up using `path` because of the `[storage.path]` key that defines a similar output directory. Happy to change it though :+1: